src/PathMaster.ts

```diff
@@ -9,7 +9,7 @@ import { Config } from "./Config";
  * If a import url isn't relative
  * and only has ascii + @ in the name it is considered a node module
  */
-const NODE_MODULE = /^([a-z@].*)$/;
+const NODE_MODULE = /^[^:]([a-z@].*)$/;
```

Comment:
Yeah I realize the dot is also for `/` e.g. `@types/foo` or even `foo/something/inside` :(